.dockerignore

```diff
@@ -1,2 +1 @@
 out/buildroot*
-
```

Comment:
remove unrealted change

Reply:
there's no reason to have a newline there but sure